src/utils/contractFns/checkProjectExist.ts

```diff
@@ -9,7 +9,9 @@ const checkProjectExist = async (
   projectId: string,
   auth: AuthContextType
 ): Promise<boolean | null> => {
-  const arcanaWeb3Instance = await getArcanaWeb3Instance(auth);
+  let arcanaWeb3Instance;
+  if (auth.isLoggedIn) arcanaWeb3Instance = await getArcanaWeb3Instance(auth);
+  else arcanaWeb3Instance = null;
```

Comment:
this can be at the time of variable declaration only.

```
let arcanaWeb3Instance = null;
if (auth.isLoggedIn) arcanaWeb3Instance = await getArcanaWeb3Instance(auth);
```